UICatalog/Scenarios/SplitContainerExample.cs

```diff
@@ -0,0 +1,149 @@
+﻿using Terminal.Gui;
+using System;
+using Terminal.Gui.Graphs;
+using NStack;
+using System.Linq;
+
+namespace UICatalog.Scenarios {
+	[ScenarioMetadata (Name: "Split Container", Description: "Demonstrates the SplitContainer functionality")]
+	[ScenarioCategory ("Controls")]
+	[ScenarioCategory ("LineView")]
+	public class SplitContainerExample : Scenario {
+
+		private SplitContainer splitContainer;
+
+		private SplitContainer splitContainer2;
+		private MenuItem miVertical;
+		private MenuItem miShowBoth;
+		private MenuItem miShowPanel1;
+		private MenuItem miShowPanel2;
+		private MenuItem miShowNeither;
+
+		/// <summary>
+		/// Setup the scenario.
+		/// </summary>
+		public override void Setup ()
+		{
+			// Scenario Windows.
+			Win.Title = this.GetName ();
+			Win.Y = 1;
+
+			Win.Add (new Label ("This is a SplitContainer with a minimum panel size of 4. Drag the splitter to resize:"));
+
+			splitContainer = new SplitContainer {
+				Y = 2,
+				X = 2,
+				Width = Dim.Fill () - 2,
+				Height = Dim.Fill () - 1,
+				SplitterDistance = Pos.Percent (50), // TODO: get this to work with drag resizing and percents
+			};
+			splitContainer2 = new SplitContainer(){
+				Width = Dim.Fill(),
+				Height = Dim.Fill(),
+				Orientation = Orientation.Horizontal
+			};
+
+			splitContainer.Panels [0].MinSize = 4;
+			splitContainer.Panels [1].MinSize = 4;
+
+			Label lbl1;
+			splitContainer.Panels [0].Title = "Hello";
+			splitContainer.Panels [0].Add (lbl1 = new Label ("Type Something:") { Y = 0 });
+			splitContainer.Panels [0].Add (new TextField () { Width = Dim.Fill (), Y = 0, X = Pos.Right (lbl1) + 1 });
+
+			Label lbl2;
+			splitContainer.Panels [1].Title = "World";
+			splitContainer.Panels[1].Add(splitContainer2);
+
+			splitContainer2.Panels [0].Add (new TextView ()
+			 {
+				Width = Dim.Fill(),
+				Height = Dim.Fill(),
+				Text = GenerateLotsOfText(),
+				AllowsTab = false,
+				WordWrap = true,
+			 });
+
+			splitContainer2.Border.BorderStyle = BorderStyle.None;
+			splitContainer2.Border.DrawMarginFrame = false;
```

Comment:
Why is this needed?

Reply:
Nevermind. I see now. 

But i also see a problem: If you want splitContainer2.Panel[1] to have a title this doesn't work.

Reply:
<img width="132" alt="image" src="https://user-images.githubusercontent.com/585482/210903769-e3039c88-b08f-4860-8f9a-a884131bdec7.png">

Reply:
With it true (and `[1].Title` set to "Two"):

<img width="136" alt="image" src="https://user-images.githubusercontent.com/585482/210903825-a5643067-7fcc-4ebe-ac10-d424e25cfd7f.png">

Reply:
So I've noticed that `BorderStyle` and `DrawMarginFrame` are tightly coupled.  If you change one but not the other then things stop laying out correctly (at least for this case).

For the tests and example I have stuck to either

**With Border**
```csharp
mySplitContainer.Border.BorderStyle = BorderStyle.Single; // the default
mySplitContainer.Border.DrawMarginFrame = true; // the default
```

**Without Border**
```csharp
mySplitContainer.Border.BorderStyle = BorderStyle.None;
mySplitContainer.Border.DrawMarginFrame = false;
```

If you do that then it does work with subpane titles.  But you get that nasty 'double nested' appearance where there are clearly 2 borders:

![image](https://user-images.githubusercontent.com/31306100/210908211-29b26b3e-50d8-4005-9ed7-e22a12330ea6.png)

Reply:
> So I've noticed that `BorderStyle` and `DrawMarginFrame` are tightly coupled. If you change one but not the other then things stop laying out correctly (at least for this case).

Yes is true. There is needed to set both, otherwise an empty frame without line is drawn.

Reply:
Resolved this because we now use `IntegratedBorder`.  `IntegratedBorder` is unique to `TileView`.  It is drawn **within** the `TileView` client area (whereas a regular border is outside) and searches child elements recursively to pull out sub elements that should be joined up into one big seamless grid and border.